src/dashboard/src/app/hsb/dashboard/page.tsx

```diff
@@ -5,6 +5,8 @@ import {
   AllocationTable,
   StorageTrendsChart,
   TotalStorage,
+  SmallBarChart,
```

Comment:
SmallBarChart looks like it should be removed

Reply:
removed